src/site/apt/troubleshooting.apt

```diff
@@ -0,0 +1,14 @@
+Troubleshooting
+
+ Standard project-build-plugin errors
+
+*-----*------*------*
+<<Error Message>>		|		<<Cause>> 		| 		<<Possible Fix>>
+*-----*------*------*
+ InvocationTargetException: No class loader avaible for bundle: ch.ivyteam.ivy.dataclasses.build    | Maven uses wrong Java version	|	Change your Java version if you have multiple versions (export JAVA_HOME=</your/Jre/Dir>)  
```

Comment:
the issue is worth to document: though I see problems with the solution
- the tip should be given in platform independent manor (windows users can write an export statement)
- and the tip walks around the real change that causes it: the driver for this error is that 7.4 runs with java 11 - while previous versions run since approx. 3 years with java 1.8.